src/main/java/uk/gov/ons/ctp/response/sample/ingest/CsvIngesterCensus.java

```diff
@@ -91,6 +91,7 @@ public SampleSummary ingest(MultipartFile file)
     SampleSummary sampleSummary;
     CensusSurveySample censusSurveySample = new CensusSurveySample();
     List<CensusSampleUnit> samplingUnitList = new ArrayList<>();
+    Integer expectedCI = 1;
```

Comment:
Why did you use Integer expectedCI = 1. Please give more details in comments about this hardcoded value